src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/Mutex.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+import 'ROOT/legacy_reputation/SafeMath.sol';
+
+
+// AUDIT: Ensure that a malicious market can't subversively cause share tokens to be paid out incorrectly.
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares
+ */
+contract ClaimProceeds is Controlled {
+    using SafeMath for uint256;
+
+    function publicClaimProceeds(IMarket _market) onlyInGoodTimes public returns(bool) {
```

Comment:
Change the visibility to `external`.  Anything that is protected by a mutex/reentrancy guard should be market as `external`.